### PR TITLE
Update grafana plugin docs

### PR DESCRIPTION
**Pull request by e-fisher**
The old data source plugin will be deprecated - updated docs with info on how to use the new k6 cloud app.
This is heavily influenced by [grafana blog](https://grafana.com/blog/2022/01/05/how-the-new-k6-cloud-app-plugin-makes-it-easy-to-correlate-qa-data-and-system-metrics-in-grafana/)

@nicolevanderhoeven  the old article had 2 links to your instruction videos which are no longer applicable - do you think we would need new videos?

EDIT: wrong person tagged

### CODE REVIEW

**Review by legander — APPROVED** [review]
Looks good, it's basically the blog post but I think it serves well as a docs page too! WDYT? @ppcano
You have my 👍  But @ppcano gets the final say.

**Comment by ppcano** [comment]
> the old article had 2 links to your instruction videos which are no longer applicable - do you think we would need new videos?

If we create the videos or any other related content, I suggest referencing them. 

At this moment, we don't have more content related to the new Grafana k6 Cloud App. Ready to go as it is.

File: src/data/markdown/docs/03 cloud/04 Integrations/06 Grafana Plugin.md [hunk 1]
```diff
@@ -3,10 +3,10 @@ title: 'Grafana Plugin'
 excerpt: 'View your k6 Cloud Test Results in Grafana'
 ---
 
-The [k6 Cloud application plugin](https://grafana.com/grafana/plugins/grafana-k6-app/) for [Grafana](https://grafana.com/grafana/) allows you to:
-* manage and visualize cloud tests to make your testing accessible in Grafana.
+The [k6 Cloud app](https://grafana.com/grafana/plugins/grafana-k6-app/) for [Grafana](https://grafana.com/) makes your testing accessible with Grafana, allowing you to:
```
Comment (ppcano) [inline src/data/markdown/docs/03 cloud/04 Integrations/06 Grafana Plugin.md:6]: @e-fisher 

On the doc, it always names it as `k6 Cloud app`, so I did an attempt to be consistent on the intro.

Still unsure. Let me know if this is inaccurate or exists a better alternative to discuss it.

Comment (e-fisher) [reply src/data/markdown/docs/03 cloud/04 Integrations/06 Grafana Plugin.md:6]: Yeah, that looks better, thank you!

**Pull request merged**